src/components/network-map-tab.js

```diff
@@ -194,78 +181,178 @@ export const NetworkMapTab = ({
         []
     );
 
+    const getEquipmentsNotFoundIds = useCallback(
+        (foundEquipmentPositions, allEquipments) => {
+            let notFoundEquipmentsIds = [];
+            const foundEquipmentsIds = Array.from(
+                foundEquipmentPositions.keys()
+            );
+            allEquipments.forEach((s) => {
```

Comment:
With `filter` and `map` (no need to initialize notFoundEquipmentsIds to []):
```
const notFoundEquipmentsIds = allEquipments
                .filter((s) => !foundEquipmentsIds.includes(s.id))
                .map((s) => s.id);
```

Reply:
done